tools/swift-ide-test/CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ add_swift_host_tool(swift-ide-test
 )
 
 # If libxml2 is available, make it available for swift-ide-test.
-if(SWIFT_HAVE_LIBXML)
```

Comment:
I think that setting `SWIFT_HAVE_LIBXML` should work for this guard.  Consider cross-compilation.